infer_structcol/main.py

```diff
@@ -91,10 +87,8 @@ class Sample:
     incident_angle: scalar
         angle of incident light on the sample
     '''
-    def __init__(self, wavelength, particle_radius, thickness, particle_index, 
+    def __init__(self, wavelength, particle_index, 
```

Comment:
After talking with barkls, we decided to keep the ranges in the inference calculation, because moving them to the Sample object wouldn't necessarily achieve anything.